app/controllers/tickets_controller.rb

```diff
@@ -28,6 +26,10 @@ def update
     end
   end
 
+  def cancell
+    render json:Tickets::CancellUnpaidTickets.new.call
```

Comment:
Typo in both -> `Cancel` instead of `Cancell`

Reply:
@wmaciejak updated

Reply:
@wmaciejak done